cpp/ps0007/150daysprogramming.cpp

```diff
@@ -26,158 +10,64 @@ float area(char shape, float dimensions[]);
 
 int main()
 {
-
-    // prompting user for shape and dimensions 
+    // prompting user for shape and dimensions 1.5/1.5
     char ch;
-    cout << "enter the shape char : \n" << "c for circle: \n r for rectangle: \n s for square: \n t for triangle: \n ";
+    cout << "enter the shape char : \n c for circle: \n r for rectangle: \n s for square: \n t for triangle: \n ";
     cin >> ch;
-    cout << area(ch, dimensions);
-
-
 
+    // printing out the area of the shape 1/1
+    float shape_area = area(ch, dimensions);
+    if (shape_area == -1)
+    {
+        cout << "invalid shape or dimensions" << endl;
+        return 1;
+    }
+    cout << shape_area;

```

Comment:
- [x] checked the user's input for choosing the shape
- [ ] filled in the dimensions array with positive numbers 
- [x] invoked function to calculate the area of the shape
- [ ] used suitable size of dimensions arrays with the shape
- [x] used `cout::std` in a right way to print out the area